README.md

```diff
@@ -535,19 +561,19 @@ The Cypress GH Action does not spawn or create any additional containers - it on
 
 ### Component tests
 
-You can run the [Cypress component tests](https://on.cypress.io/component-testing) after running E2E tests by calling the action again with a custom command:
+You can run [Cypress component tests](https://on.cypress.io/component-testing) after running E2E tests by adding `component: true`:
```

Comment:
@robertguss Updated this to clarify it would be a separate job
```suggestion
You can run [Cypress component tests](https://on.cypress.io/component-testing) in a job separate from E2E tests by adding `component: true`:
```